pinot-controller/src/test/java/org/apache/pinot/controller/helix/core/realtime/SegmentCompletionTest.java

```diff
@@ -64,7 +64,8 @@
   private final LongMsgOffset s3Offset = new LongMsgOffset(30L);
 
   @BeforeMethod
-  public void testCaseSetup() throws Exception {
+  public void testCaseSetup()
```

Comment:
changes in this file are also mainly on formats. The changes are so many that github simply folds the file up. would suggest to open a quick RP to land those format changes firstly.